fire/SFMainMod.F90

```diff
@@ -245,12 +245,11 @@ subroutine  charecteristics_of_fuel ( currentSite )
              if ( hlm_masterproc == itrue ) write(fates_log(),*) 'csa ',currentSite%acc_NI
              if ( hlm_masterproc == itrue ) write(fates_log(),*) 'sfv ',alpha_FMC
           endif
-          ! FIX(RF,032414): needs refactoring. 
-          ! average water content !is this the correct metric?         
-          timeav_swc                  = sum(currentSite%water_memory(1:numWaterMem)) / real(numWaterMem,r8)
```

Comment:
could you also remove this variable (timeav_swc) from the declarations up top?

Reply:
https://github.com/NGEET/fates/pull/561/commits/22b7f99616e1a930309eb7d9aefca2f6ced7063b#diff-c9ea8667acccc19523f68989530ca2a4R167